app/services/preserved_object_handler.rb

```diff
@@ -183,4 +184,8 @@ def string_to_int(val)
     return val.to_i if val.instance_of?(String) && val.scan(/\D/).empty?
     val
   end
```

Comment:
i think you want a corresponding `# rubocop:enable Metrics/PerceivedComplexity` here

Reply:
ew - i'm only okay with that if it's temporary and so indicated.